middlewares/build.gradle

```diff
@@ -0,0 +1,70 @@
+plugins {
+    id 'com.adarshr.test-logger' version '2.1.0'
+}
+
+dependencies {
```

Comment:
On my env, had to add ```implementation project(":core")``` to get it working.

Reply:
Yes, I know, when your middleware is merged, this will be updated too (and merge later), the current middleware code on this branch is just a sample